modules/ztp-recommended-cluster-mc-crs.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-vdu-validating-cluster-tuning.adoc
+
+:_module-type: REFERENCE
+[id="ztp-recommended-cluster-mc-crs_{context}"]
+= Recommended cluster MachineConfig CRs
+
+Check that the `MachineConfig` custom resources (CRs) that you extract from the `ztp-site-generate` container are applied in the cluster. The CRs can be found in the extracted `out/source-crs/extra-manifest/` folder.
```

Comment:
Is there anywhere in this section that tells them how to extract the "out" directory?

Reply:
I broke the content out into individual topics. But the content is unchanged except for the lines about extracting CRs form the ztp container.

I also added a link to the bottom of the content that redirects to the extract procedure.

![image](https://user-images.githubusercontent.com/74046732/203592233-46f34d81-604b-43f2-a567-a1169354bf40.png)

Reply:
https://53104--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-vdu-validating-cluster-tuning.html#ztp-recommended-cluster-mc-crs_vdu-config-ref

Reply:
@imiller0 I think this should be ready to go now. WDYT

Reply:
lgtm